autogen.pl

```diff
@@ -1096,6 +1098,7 @@ sub in_tarball {
   --no-ompi | -no-ompi          Do not build the Open MPI layer
   --no-orte | -no-orte          Do not build the ORTE layer
   --no-oshmem | -no-oshmem      Do not build the OSHMEM layer
+  --libmpi-name | -libmpi-name  Override the default lib name of: lib$libmpi_name_arg
```

Comment:
I thought it was a little odd to set it during configure time since as part of a release we want this value fixed so users don't change it. However, I could probably be convinced otherwise. Just seemed like a little bit of protection.